FinniversKit/Sources/Recycling/GridViews/AdRecommendations/Cell/JobAdRecommendationCell.swift

```diff
@@ -34,6 +34,12 @@ public class JobAdRecommendationCell: UICollectionViewCell, AdRecommendationCell
 
     private lazy var defaultImage: UIImage = UIImage(named: .noImage)
 
+    /// Extra container for accessibility issues. The cell should have "all content" and favoriteButton as accessibilty elements but it get confused if favoriteButton is a subview of the other accessibility element (so contentView can not be one of the accessibility elements
+    private lazy var containerView: UIView = {
```

Comment:
Is it possible to drop this if you define your own accessibility elements for the view?